resolwe_bio/docker_images/dnaseq/packages-debian-buildreq.txt

```diff
@@ -1,10 +1,19 @@
 build-essential
 python3-dev
-libcurl4-openssl-dev
 zlib1g-dev
 libbz2-dev
 liblzma-dev
 
+# Required for R package 'devtools'
+libcurl4-openssl-dev
```

Comment:
This package should stay where it was (above) because some other packages also depend on it.